app/models/filters/criteria/filter_for_projects.rb

```diff
@@ -8,6 +8,7 @@ def applies?
   def apply(scope)
     scope = super(scope)
     project_ids = visible_project_ids
+    return scope.none if project_ids.blank? && input.project_group_ids.present?
```

Comment:
this appears to be a bug, if a project group is provided but is empty, it is treated as thought no project group was provided.